testing_app/integration_test/app_test.dart

```diff
@@ -35,18 +35,21 @@ void main() {
 
       // Tap on the icon.
       await tester.tap(iconFinder);
+
+      // Wait 1 second for the SnackBar to be displayed
       await tester.pumpAndSettle(const Duration(seconds: 1));
 
       // Verify if appropriate message appears.
       expect(find.text('Added to favorites.'), findsOneWidget);
 
       // Tap on the icon again.
       await tester.tap(iconFinder);
+
+      // Wait 1 second for the SnackBar to be displayed
       await tester.pumpAndSettle(const Duration(seconds: 1));
 
       // Verify if appropriate message appears.
       expect(find.text('Removed from favorites.'), findsOneWidget);
-      await tester.pumpAndSettle(const Duration(seconds: 1));
```

Comment:
I think this call to `pumpAndSettle` was not necessary, the test passed after removing it